src/Grids/conformal_cubed_sphere_face_grid.jl

```diff
@@ -115,7 +115,7 @@ function ConformalCubedSphereFaceGrid(arch::AbstractArchitecture = CPU(),
     ZS = (Zᶜᶜᵃ, Zᶠᶜᵃ, Zᶜᶠᵃ, Zᶠᶠᵃ)
 
     for (ξ, η, X, Y, Z) in zip(ξS, ηS, XS, YS, ZS)
-        for i in 1:length(ξ), j in 1:length(η)
+        for i in 1:Nξ, j in 1:Nη
```

Comment:
What does this have to do with julia docs? I agree that its correct now however, and that previously it was wrong.

Reply:
It was just a note on how inbounds can be used wrong and give no error.

Reply:
It's not a problem with inbounds though right? It's a problem with the loop bounds being incorrect.

Reply:
Right